logistics/scripts/Invoke-SdkGen.ps1

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+#requires -version 5
+
+<#
+.description
+    Runs Invoke-SdkGen
+.parameter configurationFile
+    Path to the Json file containing the Integration Test Harness configuration
+.parameter apiUrl
+    API URL for the Integration Test Harness
+.parameter environmentFilePath
+    Path to environment file used to run Postman integration tests
+
+    .EXAMPLE
+    PS> Invoke-SdkGen.ps1
+#>
+param(
+    [string] $configurationFile = $(Get-RepositoryResolvedPath "logistics\scripts\smokeTestHarnessConfiguration.json"),
+    [string] $apiUrl = "http://localhost:8765",
```

Comment:
should this endpoint be pulled from the configuration file?

Reply:
No . It's defaultvalue from  in $apiUrl parameter value . Same way it's there in Invoke-PostmanIntegrationTests also .